web/src/components/njwds-extended/Heading.tsx

```diff
@@ -0,0 +1,110 @@
+import { ReactElement } from "react";
+
+export type ValidHeadingLevel = 0 | 1 | 2 | 3 | 4;
```

Comment:
While I do like `ValidHeadingLevel` since I do want to signal that those are the only allowed heading levels (or maybe `SupportedHeadingLevel`?), I see your point here. And since we don't have a need for `InvalidHeadingLevel`, I will change this. Thanks, AnJu!